lob/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 #Resources
 from lob.resource import (Address, Area, BankAccount, Check, Country, Job,
-    Letter, Object, Packaging, Postcard, Route, Service, Setting, State,
+    Letter, Object, Packaging, Postcard, Route, Setting, State,
```

Comment:
looks like theres some stray packaging code thought that was removed a while ago. think you can remove all traces of that as well?